### DISCUSSION TITLE
Connect To Another Wifi Repeater

### DISCUSSION THREAD

**Discussion by AlexKusnezov** [arendst/Tasmota#18797] [Q&A] [answered]
Hey Guys, sometimes when I restert my router or one of my reapter in the mesh network some tasmota devices will one wiath a worse connection and it's not resolved by rebooting the device.
Example:
One of my esp32 device will always connect directyl to the router which connectiuon is very weak (-83 dBm) instead of connecting to the repeater nerby which signal stregth is at elast (-50 dBm) from there.

How can I force those devices to connect to the right one?

**Comment by barbudor** [comment, marked as answer]
See SetOption56 and SetOption57

**Reply by sfromis** [reply]
If that does not help, maybe try `Reset 3` (only resets wifi calibration), and power cycle the device.
